packages/features/ee/workflows/api/scheduleSMSReminders.ts

```diff
@@ -53,83 +54,85 @@ async function handler(req: NextApiRequest, res: NextApiResponse) {
   if (!unscheduledReminders.length) res.json({ ok: true });
 
   for (const reminder of unscheduledReminders) {
-    try {
-      const sendTo =
-        reminder.workflowStep.action === WorkflowActions.SMS_NUMBER
-          ? reminder.workflowStep.sendTo
-          : reminder.booking?.smsReminderNumber;
+    if (reminder.workflowStep && reminder.booking) {
```

Comment:
Invert and continue; - you're going to deep 🙏

Reply:
Fixed, and in scheduleEmailReminder as well